azurerm/internal/acceptance/data.go

```diff
@@ -64,7 +65,7 @@ func BuildTestData(t *testing.T, resourceType string, resourceLabel string) Test
 	}
 
 	testData := TestData{
-		RandomInteger:   tf.AccRandTimeInt(),
+		RandomInteger:   AccRandTimeInt(),
```

Comment:
then this can go back to
```suggestion
		RandomInteger:   tf.AccRandTimeInt(),
```